silk-plugins/silk-plugins-rdf/src/main/scala/org/silkframework/plugins/dataset/rdf/tasks/SparqlSelectCustomTask.scala

```diff
@@ -27,7 +27,13 @@ case class SparqlSelectCustomTask(@Param(label = "Select query", value = "A SPAR
                                   limit: String = "",
                                  @Param(label = "Optional SPARQL dataset",
                                    value = "An optional SPARQL dataset that can be used for example data, so e.g. the transformation editor shows mapping examples.")
-                                  optionalInputDataset: SparqlEndpointDatasetParameter = SparqlEndpointDatasetParameter("")) extends CustomTask {
+                                  optionalInputDataset: SparqlEndpointDatasetParameter = SparqlEndpointDatasetParameter(""),
+                                  @Param(
+                                    label = "SPARQL query timeout (ms)",
+                                    value = "SPARQL query timeout (select/update) in milliseconds. A value of zero means that there is no timeout set explicitly." +
+                                        " If a value greater zero is specified this overwrites possible default timeouts.")
```

Comment:
We should use the same doc here as in SparqlDataset.scala.

Reply:
Since the SPARQL select operator is more generic, we cannot use the same description here. It just passes the timeout down to the input SPARQL endpoint, but the implementation details like default timeouts etc. depend on the SPARQL endpoint implementation and config alone.